README.md

```diff
@@ -239,36 +155,324 @@ bash install.bash
+- On Debian-based distributions, you will need to install `libcurl4` or`libcurl3` instead of `libcurl`, depending on the age of your distribution.
+
+  For example:
+
+```shell
+# Ubuntu
+sudo apt install libcurl4 openssl
+
+# Fedora
+sudo dnf install libcurl openssl
+```
+### 2. Create and activate the Virtual Environment
+
+```shell
+python3 -m venv fiftyone_env
+source fiftyone_env/bin/activate
+```
+
+Now you are ready to install **Fiftyone** and all the requirements/packages/dependencies. Go to [Installation](#-installation) section in this Readme file.
+
+</details>
+
+<details>
+<summary>MacOS</summary>
+
+<div id='-prerequisites_macos'/>
+
+### 1. Install Xcode Command Line Tools
+
+```shell
+xcode-select --install
+```
+### 2.  Install Homebrew
+```shell
+/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+```
+After you install it, follow the instructions from the Homebrew installation to set it up.
+
+### 3. Install Python and dependencies
+
+```shell
+brew install python@3.9
+brew install protobuf
+
+#optional but recommendeded
+brew install ffmpeg
+```
+### 4. Create and activate the Virtual Environment
+
+```shell
+python3 -m venv fiftyone_env
+source fiftyone_env/bin/activate
+```
+
+Now you are ready to install **Fiftyone** and all the requirements/packages/dependencies. Go to [Installation](#-installation) section in this Readme file.
+
+
+</details>
+
+<details>
+<summary>Docker</summary>
+
+<div id='-prerequisites_docker'/>
+
+Refer to
+[these instructions](https://voxel51.com/docs/fiftyone/environments/index.html#docker)
+to see how to build and run Docker images containing source or release builds
+of FiftyOne.
+
+
+</details>
+
+<div align="center">
+<p align="center">
+
+| [Windows](#-prerequisites_windows) | [Linux](#-prerequisites_linux) | [macOS](#-prerequisites_macos) | [Docker](#-prerequisites_docker) | 
+| ----------------------------------------------------------------------------- | --------------------------------------------------------------------------- | ------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------- | 
+
+</p>
+</div>
+
+**Important Notes:** Remember, you will need...
+
+-   [Python](https://www.python.org) (3.9 - 3.11)
+-   [Node.js](https://nodejs.org) - on Linux, we recommend using
+    [nvm](https://github.com/nvm-sh/nvm) to install an up-to-date version.
+-   [Yarn](https://yarnpkg.com) - once Node.js is installed, you can
+    [enable Yarn](https://yarnpkg.com/getting-started/install) via
+    `corepack enable`
+    
+</details>
+
+![------------------------------------------------------------------](https://github.com/user-attachments/assets/fb0573d0-bb56-40ff-9ae1-a5e8f62f5f42)
+
+## <img src="https://user-images.githubusercontent.com/25985824/106288517-2422e000-6216-11eb-871d-26ad2e7b1e59.png" height="20px"> **quickstart** 🚀
```

Comment:
In general, I am more in favor of capitalized headers instead of lowercase as you have it here unless we make that a more consistent branding overall.

Reply:
As we are using emojis at the start of section headers, capitalized headers appear too large and can disrupt readability.